src/common/modules/UnicodeFontHandler.js

```diff
@@ -0,0 +1,245 @@
+}
+
+/**
+ * Toggle Case.
+ *
+ * @param {string} atext
+ * @returns {string}
+ */
+function toggleCase(atext) {
+	let output = '';
+
+	for (let letter of atext) {
+		const upper = letter.toLocaleUpperCase();
+		const lower = letter.toLocaleLowerCase();
+		if (letter === lower && letter !== upper)
+			letter = upper;
+		else if (letter === upper && letter !== lower)
+			letter = lower;
+		output += letter;
+	}
+
+	return output;
+}
+
+/**
+ * Change Case
+ *
+ * @public
+ * @const
+ * @type {Object.<string, function>}
+ */
+const changeCase = {
+	"lowercase": (str) => str.toLocaleLowerCase(),
+	"uppercase": (str) => str.toLocaleUpperCase(),
+	"capitalize-each-word": (str) => capitalizeEachWord(str.toLocaleLowerCase()),
+	"toggle-case": (str) => toggleCase(str)
+};
+
+/**
+ * Handle context menu click.
+ *
+ * @param {Object} info
+ * @param {Object} tab
+ * @returns {void}
+ */
+function handle(info, tab) {
+	let text = info.selectionText;
+
+	if (text) {
+		text = text.normalize();
+		let output = '';
+
+		switch (info.menuItemId) {
+			case "lowercase":
+			case "uppercase":
+			case "capitalize-each-word":
+			case "toggle-case":
+				output = changeCase[info.menuItemId](text);
+				break;
+			case "superscript":
+			case "small-caps":
+			case "all-small-caps":
+			case "unicase":
+			case "serif-bold":
+			case "serif-italic":
+			case "serif-bold-italic":
+			case "sans-serif":
+			case "sans-serif-bold":
+			case "sans-serif-italic":
+			case "sans-serif-bold-italic":
+			case "script":
+			case "script-bold":
+			case "fraktur":
+			case "fraktur-bold":
+			case "monospace":
+			case "double-struck":
+			case "circled":
+			case "circled-(black)":
+			case "squared":
+			case "squared-(black)":
+			case "fullwidth": {
+				output = changeFont(text, info.menuItemId);
+				break;
+			}
+		}
+
+		browser.tabs.executeScript(tab.id, {
+			code: `insertIntoPage("${output}");`,
+			frameId: info.frameId
+		});
+	}
+}
+
+/**
+ * Apply new Unicode font settings.
+ *
+ * @param {Object} unicodeFont
+ * @returns {void}
+ */
+function applySettings(unicodeFont) {
+	// Thunderbird
+	const menus = THUNDERBIRD ? browser.menus : browser.contextMenus;
+
+	// menus.removeAll();
+
+	if (unicodeFont.changeCase) {
+		menus.create({
+			id: "separator-1",
+			type: "separator",
+			contexts: ["editable"]
+		});
+
+		for (const id of caseIds) {
+			// .replaceAll(" ", "-");
+			const aid = id.toLowerCase().split(" ").join("-");
+			menus.create({
+				"id": aid,
+				"title": changeCase[aid](id),
+				"contexts": ["editable"],
+			});
+		}
+	}
+
+	if (unicodeFont.changeCase && unicodeFont.changeFont) {
+		menus.create({
+			id: "separator-2",
+			type: "separator",
+			contexts: ["editable"]
+		});
+	}
+
+	if (unicodeFont.changeFont) {
+		for (const id of fontIds) {
+			if (id === "separator") {
+				menus.create({
+					// id: id,
+					type: "separator",
+					contexts: ["editable"]
+				});
+			}
+			else {
```

Comment:
```suggestion
			} else {
```

Please put that on the same line then. This makes it easier for the eye to follow that one if block is "else'ed".